ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomCommandingDemoController.swift

```diff
@@ -11,6 +11,8 @@ class BottomCommandingDemoController: UIViewController {
         view = UIView()
 
         let optionTableViewController = UITableViewController(style: .plain)
```

Comment:
why not just use maintableviewcontroller directly?

Reply:
Because the property is an optional and it's a pain dealing with optionals when configuring the view controller.